Port 443 is not open on devel.aqbanking.de connect to google.com instead.

diff -upr gwenhywfar-4.18.0.orig/checks/syncio_https.c gwenhywfar-4.18.0/checks/syncio_https.c
--- gwenhywfar-4.18.0.orig/checks/syncio_https.c	2017-11-06 11:35:39.376537746 +0100
+++ gwenhywfar-4.18.0/checks/syncio_https.c	2017-11-06 11:36:01.173317776 +0100
@@ -33,7 +33,7 @@ static int check_syncio_https1() {
 	    "ERROR in check_syncio_http1: Could not create socket io layer.\n");
     return 2;
   }
-  GWEN_SyncIo_Socket_SetAddress(sio, "devel.aqbanking.de");
+  GWEN_SyncIo_Socket_SetAddress(sio, "google.com");
   GWEN_SyncIo_Socket_SetPort(sio, 443);
 
   baseLayer=sio;
@@ -82,7 +82,7 @@ static int check_syncio_https1() {
     GWEN_DB_SetCharValue(db, GWEN_DB_FLAGS_OVERWRITE_VARS, "url", "/index.html");
 
     db=GWEN_SyncIo_Http_GetDbHeaderOut(sio);
-    GWEN_DB_SetCharValue(db, GWEN_DB_FLAGS_OVERWRITE_VARS, "Host", "devel.aqbanking.de");
+    GWEN_DB_SetCharValue(db, GWEN_DB_FLAGS_OVERWRITE_VARS, "Host", "google.com");
     GWEN_DB_SetIntValue(db, GWEN_DB_FLAGS_OVERWRITE_VARS, "Content-length", 0);
     GWEN_DB_SetCharValue(db, GWEN_DB_FLAGS_OVERWRITE_VARS, "Connection", "close");
 
